src/containers/layouts/articles/article3/article3.component.tsx

```diff
@@ -12,32 +12,39 @@ import {
 } from '@kitten/theme';
 import {
   Text,
-  Comment,
+  Comment as CommentComponent,
   CommentProps,
 } from '@src/components/common';
 import { Input } from '@kitten/ui';
-import { Article } from '@src/core/model';
+import {
+  Article,
+  Comment,
```

Comment:
would be better to import model as `CommentModel` and `Comment` as it is

Reply:
Refactored.